packages/babel-plugin-transform-simplify-comparison-operators/src/index.js

```diff
@@ -1,6 +1,25 @@
 "use strict";
 
-module.exports = function() {
+module.exports = function({ types: t }) {
+  function isPureVoid(path) {
+    return path.isUnaryExpression({ operator: "void" }) && path.isPure();
+  }
+
+  function isRealUndefined(path) {
+    return (
+      path.isIdentifier({ name: "undefined" }) &&
+      !path.scope.getBinding("undefined")
+    );
+  }
+
+  function undefinedToNull(path) {
+    if (isRealUndefined(path) || isPureVoid(path)) {
+      path.replaceWith(t.nullLiteral());
+      return true;
```

Comment:
is this return true / false needed? I don't see it used anywhere for conditional check.